src/main/kotlin/com/openlattice/auditing/LocalAuditingService.kt

```diff
@@ -0,0 +1,104 @@
+package com.openlattice.auditing
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.common.collect.ArrayListMultimap
+import com.google.common.collect.ImmutableMap
+import com.openlattice.data.DataEdge
+import com.openlattice.data.DataGraphManager
+import com.openlattice.data.EntityDataKey
+import java.util.*
+
+class LocalAuditingService(
+        private val dataGraphService: DataGraphManager,
+        private val ares: AuditRecordEntitySetsManager,
+        private val mapper: ObjectMapper
+) : AuditingManager {
+
+    fun recordEvent(event: AuditableEvent): Int {
+        return recordEvents(listOf(event))
+    }
+
+    override fun recordEvents(events: List<AuditableEvent>): Int {
+
+        val auditingConfiguration = ares.auditingTypes
+
+        return if (auditingConfiguration.isAuditingInitialized()) {
+            events
+                    .groupBy { ares.getActiveAuditEntitySetIds(it.aclKey, it.eventType) }
+                    .filter { (auditEntitySetConfiguration, _) ->
+                        auditEntitySetConfiguration.auditRecordEntitySet != null
+                    }
+                    .map { (auditEntitySetConfiguration, entities) ->
```

Comment:
Really minor thing, but "entities" is a bit confusing for me, since they are filtered events (and they have a field, that is called entities)

Reply:
+1, will update!